index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<head>
+    <title>
+        freeCodeCamp Survey Form
+    </title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1 id="title">freeCodeCamp Survey Form</h1>
+    <p id="description">
+        Thank you for
+        taking the time to help us
+        improve the platform
+    </p>
```

Comment:
@Aminocodex  I think it would be better to use "Section" for this part. It will help your code to be more readable if you use html best practices and html semantics.